### DISCUSSION TITLE
unable to change prefix

### DISCUSSION THREAD

**Discussion by Aisopia** [jagrosh/MusicBot#1255] [Questions]
I tried to change the bot prefix in config.txt (notepad++), but nope dint work. Even though other changes I made within the config file work. Did I miss something?

**Comment by Aisopia** [comment]
Nvm, I fixed it by turning on msg content intent lmao

**Reply by TalalJD** [reply]
how did you do that ?

**Reply by Woife5** [reply]
Go to the discord developer portal -> your application -> click "Bot" on the left side -> scroll down until you see "message content intent" and turn the toggle switch on
